content/vi/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+---
+title: Khám Phá Pods Và Nodes
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Mục tiêu</h3>
+                <ul>
+                    <li>Tìm hiểu Kubernetes Pods là gì ?</li>
+                    <li>Tìm hiểu Kubernetes Nodes là gì ?</li>
+                    <li>Khắc phục sự cố ứng dụng đã triển khai</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Kubernetes Pods</h2>
+                <p>Khi bạn triển khai ứng dụng thông qua Module <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes tạo ra <b>Pod</b> để lưu trữ phiên bản chạy của ứng dụng của bạn. Một Pod là một khái niệm trừu tượng của Kubernetes, đại diện cho một nhóm gồm một hoặc nhiều ứng dụng containers (ví dụ như Docker hoặc rkt) và một số tài nguyên được chia sẻ cho các containers đó. Những tài nguyên đó bao gồm:</p>
+                <ul>
+                    <li>Lưu trữ được chia sẻ, dưới dạng Tập (Volumes)</li>
+                    <li>Kết nối mạng, như một cụm địa chỉ IP duy nhất</li>
```

Comment:
`một cụm địa chỉ IP` -> `Cluster IP`
We don't need to translate some terms such as: Cluster, Node, Service, Pod,...